include/cutlass/gemm/collective/xe_array_mma.hpp

```diff
@@ -235,7 +227,7 @@ struct CollectiveMma<MainloopIntelPVCGroup<Stages, Schedule>, TileShape_, Elemen
     //
     const auto k_start_idx = crd2idx((*k_tile_iter), make_shape(K_start));
     constexpr int barrier_scope = 2;
-    int prefetch_k = 0;
+    int prefetch_k = k_start_idx;
 
     CUTLASS_PRAGMA_UNROLL
     for (; prefetch_k < DispatchPolicy::Stages; prefetch_k++) {
```

Comment:
```suggestion
    

    CUTLASS_PRAGMA_UNROLL
    for (int prefetch_k = k_start_idx; prefetch_k < DispatchPolicy::Stages; prefetch_k++) {
```

Reply:
This is not going to work because `prefetch_k` is required in the following loop as well.

Reply:
Oh I missed that you are right, there is another for loop